src/main/java/de/dhbw/karlsruhe/services/MenuDialogService.java

```diff
@@ -0,0 +1,72 @@
+package de.dhbw.karlsruhe.services;
+
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+public class MenuDialogService {
+    private Scanner scanner;
+    private String[] menuOptions;
+    private int userInput;
+    private PlayDialogService playDialogService;
+    private LeaderboardDialogService leaderboardDialogService;
+    private LogoutService logoutService;
+
+    public MenuDialogService() {
+        this.scanner = new Scanner(System.in);
+        this.menuOptions = new String[]{"Play", "Leaderboard", "Logout"};
+        this.playDialogService = new PlayDialogService();
+        this.leaderboardDialogService = new LeaderboardDialogService();
+        this.logoutService = new LogoutService();
+    }
+
+    public void startMenuDialog() {
+        this.displayMenuOptions();
+        this.userInput = this.awaitUserInput();
+        this.checkUserInput(this.userInput);
+    }
+
+    private void displayMenuOptions() {
+        System.out.println("Welcome to your favorite cli Sudoku :)");
+        for (int i=0; i < this.menuOptions.length; i++) {
+            System.out.println("[" + (i+1) + "] " + this.menuOptions[i]);
+        }
+    }
+
+    private int awaitUserInput() {
+        System.out.println("Please choose an option by entering a number!");
+        int input = -1;
+        while (input == -1) {
+            try {
+                input = this.scanner.nextInt();
+                if (!(input >=1 & input <= 3)) {
```

Comment:
Anzahl der Auswahlmöglichkeiten ist hier hardcoded. Kann mit this.menuOptions.length umgangen werden.